atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -191,7 +193,12 @@ def get_platforms(self):
     def get_current_builds(self, osbs):
         field_selector = ','.join(['status!={status}'.format(status=status.capitalize())
                                    for status in BUILD_FINISHED_STATES])
-        return len(osbs.list_builds(field_selector=field_selector))
+        try:
+            return len(osbs.list_builds(field_selector=field_selector))
+        except OsbsException:
+            self.log.exception("Error occurred while listing builds on %s" %
+                               osbs.os_conf.conf_section)
+            return self.UNREACHABLE_CLUSTER_LOAD
```

Comment:
I don't think `get_current_builds` should throw exceptions as `get_cluster_info` uses it - other functions might want some config params and ignore the fact its actually unreachable at the moment